src/Umbraco.Core/Migrations/Upgrade/V_8_7_0/ConvertToElements.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.Dtos;
+
+namespace Umbraco.Core.Migrations.Upgrade.V_8_7_0
+{
+    public class ConvertToElements : MigrationBase
```

Comment:
Would be good to have some code comments on this class and maybe renamed to ConvertToElementTypes ?

Reply:
Can do, I've got some busy projects for the next couple of days at work, but I'll add some comments when I get a chance next week.